citrination_client/models/tests/test_models_client.py

```diff
@@ -174,4 +174,22 @@ def test_can_submit_run_with_no_target():
     run = _trigger_run(client, view_id, target=None)
 
     assert_run_accepted(view_id, run, client)
-    kill_and_assert_killed(view_id, run, client)
\ No newline at end of file
+    kill_and_assert_killed(view_id, run, client)
+
+@pytest.mark.skipif(environ['CITRINATION_SITE'] != "https://citrination.com", reason="Data view status tests only supported on open")
+def test_data_view_status_reports_services_ready():
+    """
+    Tests that a data view on open citrination which has successfully trained reports
+    the status of it's services as ready.
+    """
+
+    view_id = "524"
+
+    status = client.get_data_view_service_status(data_view_id=view_id)
```

Comment:
Can we add this call and the following asserts to a few other of the tests (like retrain) to make sure it works for a non-ready status?

Reply:
@maxhutch `retrain` is in the internal SDK, but not in PyCC. Unfortunately PyCC has no functionality yet for triggering model retrains (other than adding more data to an underlying data view). We could find a permanently broken view though.

Reply:
Then let's just make a note to use this in the internal SDK once its merged.

Reply:
Sounds good.